force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -227,19 +310,33 @@ public with sharing class ClassesObjectsMethods {
      */
     public class Book {
         // Declare three public instance variables - title, author, pages
-        // title
-        // author
-        // pages
+        
+        public String title {
+            get { return title;}
+            set { title = value;}
+        }
+        public String author {
+            get { return author;}
+            set { author = value;}
+        }
+        public Integer pages {
+            get { return pages;}
+            set { pages = value;}
+        }
     
         // Book constructor to initialize the title and author
-        public Book() {
+        public Book(string title, string author) {
             // Initialize the title and author
+            this.title = title;
+            this.author = author;
```

Comment:
You are the first student I have ever seen use this syntax for the getters and setters. Nice work.  One of the nice things about Apex vs Java is we don't have to write these out. 

I the real world I pretty much only use that syntax to make a variable public get and private set

```
public String author {get; private set;};
````